pages/api/mock/[...slug].ts

```diff
@@ -121,24 +136,7 @@ export default async function handler(
   const org_id = Number(req.headers["org_id"]);
   const slug = queryParams.slug as string[];
   delete queryParams.slug;
```

Comment:
What if there are parameters in the url itself? E.g. in `/api/v1/customer/{id}`?

Edit: have seen line 174 which does this - perhaps add a comment explaining path parameters are dealt with below?